fem/eltrans.cpp

```diff
@@ -351,19 +353,50 @@ int InverseElementTransformation::Transform(const Vector &pt,
          const int order = std::max(T->Order()+rel_qpts_order, 0);
          if (order == 0)
          {
-            ip0 = &Geometries.GetCenter(T->GetGeometryType());
+            ip0 = Geometries.GetCenter(T->GetGeometryType());
          }
          else
          {
             RefinedGeometry &RefG = *refiner.Refine(T->GetGeometryType(), order);
             int closest_idx = (init_guess_type == ClosestPhysNode) ?
                               FindClosestPhysPoint(pt, RefG.RefPts) :
                               FindClosestRefPoint(pt, RefG.RefPts);
-            ip0 = &RefG.RefPts.IntPoint(closest_idx);
+            ip0 = RefG.RefPts.IntPoint(closest_idx);
+         }
+         break;
+      }
+      case EdgeScan:
+      {
+         const int order = std::max(T->Order() + rel_qpts_order, 0);
+         if (order == 0)
+         {
+            ip0 = Geometries.GetCenter(T->GetGeometryType());
+         }
+         else
+         {
+            auto &ir = *refiner.EdgeScan(T->GetGeometryType(), order + 1);
+            int res = Outside;
+            int npts = ir.GetNPoints();
+            // will only return Outside if all test points report Outside
```

Comment:
This comment was a bit misleading. 
Looking at the loop below, it might be more accurate to say
```suggestion
            // will return Inside if any test point reports Inside, Outside if all points report Outside, else Unknown
```